TesseractCore/Edge.swift

```diff
@@ -24,6 +24,12 @@ public struct Edge: Hashable {
 	}
 }
 
+extension Edge: Printable {
+    public var description: String {
+        return "\(source.identifier): \(source.outputIndex) -> \(destination.identifier): \(destination.inputIndex)"
+    }
+}
```

Comment:
1. Tabs.
2. Could you add this to the definition of `Edge` instead of adding it in an extension, please?